index.html

```diff
@@ -594,8 +594,8 @@ <h4>parseBaseProofValue</h4>
 The following algorithm parses the components of a `bbs-2023` selective
 disclosure base proof value. The required input is a proof value
 (<var>proofValue</var>). A single object, <em>parsed base proof</em>, containing
-three elements, using the names "bbsSignature", "hmacKey",
-and "mandatoryPointers", is produced  as output.
+three elements, using the names "bbsSignature", "bbsHeader", "publicKey",
+"hmacKey", and "mandatoryPointers", is produced  as output.
```

Comment:
It was 3 named elements; now it's 5 named elements...
```suggestion
five elements, using the names "bbsSignature", "bbsHeader", "publicKey",
"hmacKey", and "mandatoryPointers", is produced  as output.
```